app/service/ccd-service.ts

```diff
@@ -135,6 +140,24 @@ class CcdService {
     return updatesCase;
   }
 
+  async submitAppeal(userId: string, updatedCase: CcdCaseDetails, headers: SecurityHeaders): Promise<CcdCaseDetails> {
+    const updateEventResponse = await this.startUpdateCase(userId, updatedCase.id, Events.SUBMIT_APPEAL, headers);
+    logger.trace('Submitting appeal case', logLabel);
+
+    const updatesCase = await this.submitUpdateCase(userId, updatedCase.id, headers, {
+      event: {
+        id: updateEventResponse.event_id,
+        summary: 'Submit Update Appeal case AIP',
+        description: 'Submit Update Appeal case AIP'
+      },
+      data: updatedCase.case_data,
+      event_token: updateEventResponse.token,
+      ignore_warning: true
+    });
+
+    return updatesCase;
+  }
+
   async loadOrCreateCase(userId: string, headers: SecurityHeaders): Promise<CcdCaseDetails> {
```

Comment:
Merge updateCase and submitAppeal into 'updateAppeal' method passing the event type